src/candy-machine/candy-machine.service.ts

```diff
@@ -205,203 +238,275 @@ export class CandyMachineService {
           },
         });
 
-      const newCollectionNft = await createCollectionNft(
-        this.metaplex,
-        onChainName,
-        collectionNftUri,
-        comicIssue.sellerFeeBasisPoints,
-      );
+      if (tokenStandard == TokenStandard.Core) {
+        const collection = generateSigner(umi);
+        const collectionBuilder = createCollectionV1(umi, {
+          collection,
+          uri: collectionNftUri,
+          name: onChainName,
+        });
+
+        await collectionBuilder.sendAndConfirm(umi, {
+          send: { commitment: 'confirmed' },
+        });
+
+        console.log(`Collection: ${collection.publicKey.toString()}`);
```

Comment:
intentional to leave console log here?

Reply:
yep, i'm logging these details specifically for our use. This is admin use function and many times tx failure or different issues like failure to confirm tx we lose data to check onchain and recover these.

Reply:
That's okay, I was curios.